libs/blocks/tabs/tabs.js

```diff
@@ -95,8 +95,7 @@ function changeTabs(e) {
   content
     .querySelectorAll(`.tabpanel[data-block-id="${blockId}"]`)
     .forEach((p) => p.setAttribute('hidden', true));
-
-  content.querySelector(`#${target.getAttribute('aria-controls')}`).removeAttribute('hidden');
+  targetContent.removeAttribute('hidden');
```

Comment:
```suggestion
  targetContent?.removeAttribute('hidden');
```
The value is based on some conditionals and query selectors which might not yield results, let's play safe with this one.